backend/FlowersApi/Models/Customer/CreateCustomerDto.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace FlowersApi.Models.Customer
+{
+    public class CreateCustomerDto
+    {
+        [Required]
+        public string? UserName { get; set; }
+
+        public string? Password { get; set; }
```

Comment:
do we need password if we use auth0 for login?

Reply:
will be left as is, if needed later